reference/core/tls.md

```diff
@@ -8,7 +8,48 @@ The Ambassador Edge Stack's robust TLS support exposes configuration options for
 - [Server Name Indication (SNI)](../../../user-guide/sni)
 - [TLS Origination](../../tls/origination)
 
-In addition, users of the Ambassador Edge Stack can take advantage of the [Host CRD](/reference/host-crd) to expose the Ambassador Edge Stack. 
+## `Host`
+
+The `Host` resource has been added to the Ambassador API Gateway and Ambassador Edge Stack in version 1.0.0.
+
+The `Host` resource simplifies TLS by directly connecting how Ambassador performs TLS termination and the domains it is serving. With the Ambassador Edge Stack, the `Host` can issue and manage certificates automatically using the ACME protocol. 
+
+For both the Ambassador Edge Stack and API Gateway, it can read a certificate from a Kubernetes secret and use that certificate to terminate TLS on a domain.
```

Comment:
the `Host` can read

Reply:
"Can read does not feel right since, when Ambassador is doing ACME, it is actually doing a lot more than reading the certificates from a secret. It is creating and managing the certificates as well as reading them.

Reply:
Okay - I'm not sure what the "it" is referring to on line 17. My understanding was that it was the Host, but if that's not it, we should specify what it is.

Reply:
Ahhh I was confused. It is  clear you were right 🙂 Thanks!